vstc/Makefile

```diff
@@ -0,0 +1,18 @@
+VENDOR=Spirent
+NAME=vstc
+IMAGE_FORMAT=qcow2
+IMAGE_GLOB=*.qcow2
+
+VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\+\.[0-9]\+\.[0-9]\+[sb]\?\?\)\([^0-9].*\|$$\)/\1/')
```

Comment:
@kaelemc let's make our lifes easier and ask a user to rename the qcow to a universal and easily parseable image name
like `spirent_vstc-<any version string>.qcow2`